.github/workflows/workflow.yml

```diff
@@ -0,0 +1,11 @@
+name: basic tests
+
+on: push
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - run: exit 0
```

Comment:
Could you just add a `flake8` command? So that at least we lint the code from day 1